src/Infrastructure.Dapper/AdminConsole/Repositories/OrganizationRepository.cs

```diff
@@ -251,4 +262,103 @@ await connection.ExecuteAsync("[dbo].[Organization_IncrementSeatCount]",
             new { OrganizationId = organizationId, SeatsToAdd = increaseAmount, RequestDate = requestDate },
             commandType: CommandType.StoredProcedure);
     }
+
+    public async Task AddUsersToPasswordManagerAsync(
+        Guid organizationId,
+        DateTime requestDate,
+        Plan plan,
+        IEnumerable<CreateOrganizationUser> organizationUserCollection)
+    {
+        var organizationUsersList = organizationUserCollection.ToList();
+
+        await using var connection = new SqlConnection(ConnectionString);
+
+        await using var transaction = await connection.BeginTransactionAsync();
```

Comment:
ℹ️ Speak with @rkac-bw about what isolation level we need to use here. SQL Azure defaults now have uncommitted reads so this won't work.